### ISSUE TITLE
Сhaos rifts wrong logic

### ISSUE THREAD

**Issue by cmd-balboa** [beyond-aion/aion-server#13]
The wrong logic of the chaos rifts. We need some good NPCs to appear depending on the portal. Also, we need to be able to control them with a schedule. In the chaos rift, some NPCs become others with good rewards but strong ones. I know that the chaos rifts open on different days.

**Comment by Estrayl** [comment]
Chaos rifts are controlled like any other rift

See:
[RiftSchedule](https://github.com/beyond-aion/aion-server/blob/4.8/game-server/config/schedule/rift_schedule.xml)
[RiftSpawnsEnshar](https://github.com/beyond-aion/aion-server/blob/4.8/game-server/data/static_data/spawns/Rifts/220080000_Enshar.xml)

We know of a specific problem where the standard version of mobs does not despawn and overlaps with the chaos version. To fix this, rift spawns need similar logic to siege spawns, where we can declare different states.

**Comment by neon-dev** [comment]
@cmd-balboa please do not close your issues unless they are fully resolved.

**Comment by determinated** [comment]
Indeed, after checking it for a while, those rifts have wrong logic/don't work properly. The boss honestly for some reason don't even spawn. Mobs are spawned on the same location as normal current mobs, don't know if the mechanic would replace normal mobs for volatille mobs. It's honestly one of the most important 4.8 mechanics for resources.

**Comment by Estrayl** [comment]
The "Chaos Rift" can spawn in one of three elite zones on the map. It leads to the corresponding elite zone on the other map.
The rift boss spawns at the barrier of the exit location. Chaos mobs will continue to spawn in the other two elite zones.

Currently, the only issue is the missing despawn of the "peaceful" version of the Chaos mobs. The feature itself is fully accessible.
